qdrant-landing/content/documentation/concepts/filtering.md

```diff
@@ -944,13 +1089,28 @@ client
 ```
 
 ```csharp
-using Qdrant.Client;
-using Qdrant.Client.Grpc;
-using static Qdrant.Client.Grpc.Conditions;
+import (
+	"context"
 
-var client = new QdrantClient("localhost", 6334);
+	"github.com/qdrant/go-client/qdrant"
+)
 
-await client.ScrollAsync(collectionName: "{collection_name}", filter: MatchKeyword("country.name", "Germany"));
+client, err := qdrant.NewClient(&qdrant.Config{
+	Host: "localhost",
+	Port: 6334,
+})
+
+client.Scroll(context.Background(), &qdrant.ScrollPoints{
+	CollectionName: "{collection_name}",
+	Filter: &qdrant.Filter{
+		Should: []*qdrant.Condition{
+			qdrant.NewMatch("country.name", "Germany"),
+		},
+	},
+})
+```
+
+```go
 ```
```

Comment:
Here the Go sample is in the csharp section (and the original C# is dropped)

Reply:
Thanks. Will fix.